### PR TITLE
Fix style issues in infrastructure tests

### PR DESCRIPTION
**Pull request by bjss-chris**
Activity: [AB#9815](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/9815)
Tasks: [AB#11094](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11094), [AB#11095](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11095)

### CODE REVIEW

File: tests/NHSD.BuyingCatalogue.Infrastructure.Tests/EnumeratorTests.cs [hunk 1]
```diff
@@ -62,12 +61,10 @@ public void FromValueGivenValueSmallShouldBeEqual()
         [Test]
         public void FromValueGivenValueFourShouldThrow()
         {
-            var expectedValue = 4;
+            const int expectedValue = 4;
```
Comment (PhysicsJon) [inline tests/NHSD.BuyingCatalogue.Infrastructure.Tests/EnumeratorTests.cs:64]: Should this have a capitalised first letter?

File: tests/NHSD.BuyingCatalogue.Infrastructure.Tests/EnumeratorTests.cs [hunk 2]
```diff
@@ -81,12 +78,10 @@ public void FromNameGivenNameMediumShouldBeEqual()
         [Test]
         public void FromNameGivenNameExtraLargeShouldThrow()
         {
-            var expectedName = "extra large";
+            const string expectedName = "extra large";
```
Comment (PhysicsJon) [inline tests/NHSD.BuyingCatalogue.Infrastructure.Tests/EnumeratorTests.cs:81]: Should this have a capitalised first letter?

**Review by PhysicsJon — CHANGES_REQUESTED** [review]

File: tests/NHSD.BuyingCatalogue.Infrastructure.Tests/EnumeratorTests.cs (hunk 1 above)
Comment (bjss-chris) [reply tests/NHSD.BuyingCatalogue.Infrastructure.Tests/EnumeratorTests.cs:64]: Not within a method, only instance level constants should start with a cap.

File: tests/NHSD.BuyingCatalogue.Infrastructure.Tests/EnumeratorTests.cs (hunk 2 above)
Comment (bjss-chris) [reply tests/NHSD.BuyingCatalogue.Infrastructure.Tests/EnumeratorTests.cs:81]: As above.

**Review by PhysicsJon — APPROVED** [review]

File: tests/NHSD.BuyingCatalogue.Infrastructure.Tests/EnumeratorTests.cs (hunk 1 above)
Comment (PhysicsJon) [reply tests/NHSD.BuyingCatalogue.Infrastructure.Tests/EnumeratorTests.cs:64]: That's fair, approved

File: tests/NHSD.BuyingCatalogue.Infrastructure.Tests/EnumeratorTests.cs (hunk 2 above)
Comment (PhysicsJon) [reply tests/NHSD.BuyingCatalogue.Infrastructure.Tests/EnumeratorTests.cs:81]: That's fair, approved

**Pull request merged**